nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -245,6 +245,7 @@ planner_server:
   ros__parameters:
     planner_plugin_types: ["nav2_navfn_planner/NavfnPlanner"]
     planner_plugin_ids: ["GridBased"]
+    compute_plan_warn_duration_ms: 100.0
```

Comment:
I think this should be analog to the controller_server's `controller_frequency`. I think then `expected_planner_frequency`.

Whatever the name, as a new parameter, we need to add it to https://github.com/ros-planning/navigation2/blob/master/doc/parameters/param_list.md and https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-planner-server.rst as well as the migration guide that the change exists

Reply:
> I think this should be analog to the controller_server's `controller_frequency`. I think then `expected_planner_frequency`.

@SteveMacenski On second thought I'm not sure if **expected frequency** will be the right parameter. There's not present a loop inside the planner server (There is?). In these case, max_elapsed_time sounds better. What do you think?